mobile-widgets/qml/HintsTextEdit.qml

```diff
@@ -14,7 +14,11 @@ TextField {
 	onTextChanged: {
 		textUpdateTimer.restart();
 	}
-	onFocusChanged: frame.shouldShow = focus
+	onFocusChanged: {
+		focus != focus
```

Comment:
if I do "focus = !focus" the app crashes hard.
on the other hand after commit 12d86e this is not needed so I merged the change into that.